zap/src/main/java/org/zaproxy/zap/extension/pscan/PassiveScanTaskHelper.java

```diff
@@ -168,8 +168,20 @@ public void raiseAlert(HistoryReference href, Alert alert) {
      * Adds the given tag to the specified message.
      *
      * @param tag the name of the tag.
+     * @deprecated 2.12.0 Replaced by {@link #addHistoryTag(HistoryReference, String)}
```

Comment:
This is a new class, the "old" method can be renamed.

Reply:
Okay, thanks